libs/SalesforceHybridSDK/SalesforceHybridSDKTestApp/Classes/AppDelegate.m

```diff
@@ -197,15 +189,20 @@ - (void)initializeAppViewState
     self.window.rootViewController = self.viewController;
     [self.window makeKeyAndVisible];
 }
-                                                            
 
-@end
+- (void)logWithLevel:(DDLogLevel)level format:(NSString *)format, ... {
```

Comment:
You had to introduce such a method in many places.
Could we have a default logger for all those cases?